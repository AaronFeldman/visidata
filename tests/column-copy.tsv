Rep	sum_Total	adj_sum_Total
Jones	2363.04	2000.01
Jardine	2812.19	2812.19
Gill	1749.87	1749.87
Kivell	3109.44	3109.44
Sorvino	1283.61	1283.61
Andrews	438.37	438.37
Morgan	1387.77	1387.77
Parent	3102.30	3102.30
Smith	1641.43	1641.43
Thompson	1203.11	1203.11
Howard	536.75	536.75
