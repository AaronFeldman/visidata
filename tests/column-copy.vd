sheet	col	row	keystrokes	input	comment
		0	o	sample_data/sample.tsv	open file
sample	Total	0	%		set column type to float
sample	Total	0	+	sum	choose aggregator for this column
sample	Rep	0	F		open frequency table from values in this column
sample_Rep_freq	sum_Total	0	e	2000	edit this cell [Exception: column cannot be changed]
sample_Rep_freq	sum_Total	0	'		add a frozen copy of this column
sample_Rep_freq	sum_Total_frozen	0	^	adj_sum_Total	rename this column
sample_Rep_freq	adj_sum_Total	0	e	2000.01	edit this cell
