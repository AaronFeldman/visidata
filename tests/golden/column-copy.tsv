Customer	count	Paid_sum	adj_sum_Paid
Robert Armstrong	2	2,331.00	2,000.00
Susan Ashworth	1	4,084.27	4,084.27
桜 高橋 (Sakura Takahashi)	1	785.00	785.00
Kyle Kennedy	1	1,260.66	1,260.66
David Attenborough	1	59.43	59.43
Douglas "Dougie" Powers	1	36.04	36.04
Rubeus Hagrid	1	3,827.10	3,827.10
Juan Johnson	1	191.39	191.39
Monica Johnson	1	4.98	4.98
Helen Halestorm	1	65.88	65.88
Jon Arbuckle	1	242.45	242.45
Wil Wheaton	1	39.95	39.95
María Fernández	1	4.00	4.00
Mr. Praline	1	-2,300.00	-2,300.00
Michael Smith	1	31.85	31.85
